packages/barcode-scanning/android/src/main/java/io/capawesome/capacitorjs/plugins/mlkit/barcodescanning/BarcodeScannerHelper.java

```diff
@@ -191,4 +383,10 @@ private static Point[] normalizeCornerPoints(@NonNull Point[] cornerPoints, @Non
         // Logger.debug("Normalized corner points: " + normalizedCornerPoints[0] + ", " + normalizedCornerPoints[1] + ", " + normalizedCornerPoints[2] + ", " + normalizedCornerPoints[3]);
         return normalizedCornerPoints;
     }
+
+    private static void putIfNotNull(JSObject result, String key, Object value) {
+        if (value != null) {
+            result.put(key, value);
+        }
+    }
```

Comment:
This method is actually not necessary, because `null` values are not returned anyway. If you want to return a `null` value, then you would have to do the following:

```java
result.put("key", value == null ? JSONObject.NULL : value);
```

So you should be able to just call `result.put(key, value)`.